tests/functional/filter/cram/subsample-group-by-without-force-included-strains.t

```diff
@@ -0,0 +1,16 @@
+Setup
+
+  $ pushd "$TESTDIR" > /dev/null
+  $ source _setup.sh
+
+Do not consider force-included strains for subsampling.
+In this test, we force-include two old strains that are the only representatives of their month/year date group (December 2015).
+We don't filter these strains, so they could be considered for subsampling, but Augur removes them from consideration because they have been force-included.
+
+  $ ${AUGUR} filter \
+  >   --metadata filter/data/metadata.tsv \
+  >   --include filter/data/include_old_strains.txt \
```

Comment:
I'd prefer if the `--include` file was just created in this file such as:

https://github.com/nextstrain/augur/blob/5def1a7284e181b8a350221e34f6988dda40c795/tests/functional/filter/cram/filter-force-include-no-duplicates.t#L12:L18